x-pack/plugins/apm/public/components/shared/charts/TransactionCharts/index.tsx

```diff
@@ -101,11 +101,13 @@ export class TransactionCharts extends Component<TransactionChartProps> {
       return null;
     }
 
-    const { serviceName, transactionType, kuery } = this.props.urlParams;
+    const { serviceName, kuery } = this.props.urlParams;
     if (!serviceName) {
       return null;
     }
 
+    const linkedJobId = ''; // TODO link to ML job id for the selected environment
```

Comment:
Good idea with TODO's throughout.
Maybe you can call them `ML TODO: ...`  to make it easier to search for these specific TODOs when re-implementing ml integration